components/browser/engine-gecko-beta/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineView.kt

```diff
@@ -38,9 +38,14 @@ class GeckoEngineView @JvmOverloads constructor(
             } catch (e: IllegalStateException) {
                 // This is to debug "display already acquired" crashes
                 val otherActivityClassName =
-                    this.session?.accessibility?.view?.context?.javaClass?.simpleName
+                        this.session?.accessibility?.view?.context?.javaClass?.simpleName
+                val otherActivityClassHashcode =
+                        this.session?.accessibility?.view?.context?.javaClass?.hashCode()
                 val activityClassName = context.javaClass.simpleName
-                val msg = "ATTACH VIEW: Current activity: $activityClassName Other activity: $otherActivityClassName"
+                val activityClassHashCode = context.javaClass.hashCode()
```

Comment:
@pocmo is `context.javaClass.hashCode()`  the right `hashCode` ? I'm a bit confused if this is the `hashCode` for the `Class` object or if this is the one for the activity object?

Reply:
Oh, I missed that. No, we want the hashCode of the context instances here! :)

Reply:
Thanks :)